_includes/configuration-db.md

```diff
@@ -1,5 +1,3 @@
-<div class="important">
```

Comment:
@meghasalunke1 was removing this intentional?

Reply:
Yes, Adam because in v3.5.0 documentation, this topic is not appearing. I checked other topics to understand what's wrong, then noticed these extra tags were causing the problem. Removing this should fix the issue.